index.src.html

```diff
@@ -1901,6 +1907,29 @@
   </dd>
 </dl>
 
+Latency Mode{#latency-mode}
+---------------------------
+<xmp class='idl'>
+enum LatencyMode {
+  "quality",
+  "realtime"
+};
+</xmp>
+
+: <dfn enum-value for=LatencyMode>quality</dfn>
+:: Indicates that the user agent should optimize for encoding quality. In
+    this mode:
+    * User agents may increase encoding latency to improve quality.
```

Comment:
We'll want to do one pass to capitalize the rfc2119 words I think at some stage. There are lots of uses in non-normative section also.

Reply:
Sg, filed #301

Reply:
Filed #301